sawtooth_explainer.md

```
@@ -0,0 +1,49 @@
+    
+  1. two adjacent TAILS also share 99.9% of compute 
+     1. `[1:00 7 days ago, 1:00)` vs. `[1:05 7 days, 1:05]`  
+  
+  
+## Sawtooth aggregator
+
+we exploit these properties to aggressively reuse the partial sums
+there are three layers
+
+1. compute all tile sums (one per 5m - `1:00 - 1:05`, `1:05 - 1:10`, ...)
+  1. `key, [event]` => `key, [tile_ir]` (combine reduce)
```

markdownlint-cli2 (0.22.1)

[warning] Inconsistent indentation for list items at the same level
Expected: 0; Actual: 2

(MD005, list-indent)